src/components/charts/heatMap3DPlot/MultiAxisScatter.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import ReactEcharts from 'echarts-for-react';
+import PropTypes from 'prop-types';
+import 'echarts';
+import 'echarts-gl';
+
+class MultiAxisScatter extends React.PureComponent {
```

Comment:
I think this is the one to keep. Right?

Reply:
Yes, this is the one that is being used.